build/gen_stub.php

```diff
@@ -5351,7 +5711,12 @@ function(?ArgInfo $aliasArg, ?ArgInfo $aliasedArg) use ($aliasFunc, $aliasedFunc
 }
 
 if ($generateMethodSynopses) {
-    $methodSynopsesDirectory = getcwd() . "/methodsynopses";
```

Comment:
hmm, maybe it would be more clear to just require a target directory, just like in case of the replace options... IMO being backward compatible is not super important.

Reply:
A problem is when you only want to generate the methodsynopsis for one extension, but you need to run gen_stub for additional directories.. E.g. you need `Zend/` for `ext/json`, since `JsonException` is inherited from `Exception`, etc. In this case, the additionally "required" symbols will also be generated... Which is not ideal, and I'm not sure about a good solution

Reply:
They would get generated but not replacing the existing page. So I don't really see the issue here.

If you explain me where the target dir $arg would come from I can add it, but I couldn't figure this out so I just took the manual, as that's what I expect it to do frankly